dominion/templates/country_page.html

```diff
@@ -21,15 +21,12 @@
   {% include 'blocks/partners.html' %}
 {% endblock content_container %}
 
-{% block body_javascript_extra %}
+{% block homepage_javascripb_libs %}
```

Comment:
I think `homepage_javascripb_libs` is only used to _include_ libs and not write custom JS. That's why this code was put in the more general `body_javascript_extra`. Did it cause any errors?

Reply:
Khadija changed this due to map issues when it comes to drawing. @KhadijaMahanga might better explain this.